src/main/java/tqs/example/impostor/service/AdminService.java

```diff
@@ -0,0 +1,60 @@
+package tqs.example.impostor.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import tqs.example.impostor.repository.*;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Service
+public class AdminService implements AdminServiceInterface{
```

Comment:
Admin service should implement all methods that are necessary for all CRUD functionalities. For now the only functionality that is implemented is reading from db.